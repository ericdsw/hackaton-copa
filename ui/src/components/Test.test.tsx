import * as React from 'react';
import { shallow } from 'enzyme';
import initStoryshots from '@storybook/addon-storyshots';

import Test from './Test';

describe('<Test />', () => {
  initStoryshots({
    suite: 'Serialized Snapshots',
    storyKindRegex: /^Test$/,
    renderer: shallow,
  });
});
