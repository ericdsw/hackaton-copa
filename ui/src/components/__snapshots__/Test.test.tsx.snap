// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Test /> Serialized Snapshots Test default 1`] = `ShallowWrapper {}`;
